api/src/main/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExhaustionController.java

```diff
@@ -105,6 +112,9 @@ private void handleExhaustedUsers(Set<DbUser> newlyExhaustedUsers) {
           initialCreditsService.updateInitialCreditsExhaustion(user, true);
           // delete apps and runtimes
           deleteAppsAndRuntimesInInitialCreditsWorkspaces(user);
```

Comment:
I removed `deleteAppsAndRuntimesInInitialCreditsWorkspaces` here and incorporated it into `updateInitialCreditsExhaustion` within InitialCreditsService. This now calls `stopInitialCreditSpendInWorkspace` where it both deletes apps and runtimes and unlinks billing.

I moved this logic into the InitialCreditsService, because it seems like the exhausted and expired workflows had separate implementations of this same logic. Moving it into InitialCreditsService lets the two share.

What do you think? I am open to a different approach.